internal/provider/kubernetes/config/default/auth_proxy_patch.yaml

```diff
@@ -27,8 +27,3 @@ spec:
           requests:
             cpu: 5m
             memory: 64Mi
-      - name: envoy-gateway
-        args:
-        - "--health-probe-bind-address=:8081"
-        - "--metrics-bind-address=127.0.0.1:8080"
-        - "--leader-elect"
```

Comment:
Why are these changes required?

Reply:
these args were being appended to the `envoy-gateway` container, which I dont think is the intent